lib/parent-require.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (id) {
```

Comment:
don't forget to credit the original author with a code comment at the top of the file https://github.com/jaredhanson/node-parent-require